tests/test_decision_maker/test_base.py

```diff
@@ -265,6 +266,7 @@ def setup_class(cls):
         cls.preferences = Preferences()
         cls.decision_maker = DecisionMaker(agent_name=cls.agent_name, max_reactions=MAX_REACTIONS, outbox=cls.outbox,
                                            wallet=cls.wallet, ledger_apis=cls.ledger_apis)
+        cls.multiplexer.connect()
```

Comment:
again, this is needed because now the `out_queue` (called below by `outbox.empty()`) is not initialized unless the multiplexer set up the connections.

Reply:
hmm ok